### ISSUE TITLE
DrillDown with groupded bar chart

### ISSUE THREAD

**Issue by vnanaware** [highcharts/highcharts-ios#88]
I want to use drilldown bar chart when I click on bar chart, Then wants to display grouped bar chart. How to do this?

**Comment by ihnatmoisieiev** [comment]
Hello @vnanaware,

do you have any demo in `JS`?

**Comment by vnanaware** [comment]
No,     Tell me one thing is it possible to get selected index of drilldown column?   right now I getting null

Below is the my code 
let plotOptions=HIPlotOptions()
        plotOptions.series=HISeries()
        plotOptions.series.dataLabels=HIDataLabels()
        plotOptions.series.dataLabels.enabled=true
        plotOptions.series.dataLabels.format="{point.y:.1f}%"
        plotOptions.series.allowPointSelect=true
        plotOptions.series.point=HIPoint()
        plotOptions.series.point.events=HIEvents()
        plotOptions.series.point.events.click = HIFunction(closure: { (context) in
            guard let context = context else { return }
   
            print("Selected series index: \(context.getProperty("index")!)")
            
        },properties: ["index"])

**Comment by vnanaware** [comment]
![simulator screen shot - iphone x - 2018-04-20 at 14 46 02](https://user-images.githubusercontent.com/16416191/39042883-9fbc5f8a-44a9-11e8-8fbb-c6e07410e365.png)

**Comment by vnanaware** [comment]
Above is the my column with drilldown chart. I want to selected index of above column

**Comment by ihnatmoisieiev** [comment]
@vnanaware did you try it? Can I close the issue?
